### PR TITLE
fix(hepa): fix key-auth for mse and nginx check for kong

### PR DESCRIPTION
**Pull request by wangzhuzhen**
#### What this PR does / why we need it:
Fix 3 bugs：
(1) Lack of credential for default config for mse key-auth plugin.
(2) MSE gateway  redirect type url  support modify
(3) Nginx custom config conflict check

#### Specified Reviewers:

/assign @dspo @luobily 

#### ChangeLog
Bugfix： Fix the bug that some configurations for MSE or Nginx-Kong Gateway（修复了 MSE key-auth 和 Kong 自定义 Nginx 配置校验的问题）

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |      Fix the bug that some configurations for MSE or Nginx-Kong Gateway        |
| 🇨🇳 中文    |     修复了 MSE 和 Kong 自定义 Nginx 配置校验的问题         |

#### Need cherry-pick to release versions?

/cherry-pick release/2.4-beta.1

### CODE REVIEW

File: internal/tools/orchestrator/hepa/services/api_policy/impl/impl.go [hunk 1]
```diff
@@ -1385,7 +1385,12 @@ func hasDuplicatedConfig(p1, p2 apipolicy.PolicyConfig) (string, error) {
 
 	for k := range p1_Nginx {
 		if val, ok := p2_Nginx[k]; ok {
-			return k + fmt.Sprintf("=%s", val), nil
+			// more_set_headers、proxy_set_header、set、limit_req、limit_conn、error_page、deny、allow、return 等允许多次设置
+			if k != "more_set_headers" && k != "proxy_set_header" &&
+				k != "set" && k != "limit_req" && k != "limit_conn" &&
+				k != "error_page" && k != "deny" && k != "allow" && k != "return" {
+				return k + fmt.Sprintf("=%s", val), nil
+			}
```
Comment (luobily) [inline internal/tools/orchestrator/hepa/services/api_policy/impl/impl.go:1393]:
```suggestion
skipKeys := map[string]bool{
    "more_set_headers": true,
    "proxy_set_header": true,
    "set": true,
    "limit_req": true,
    "limit_conn": true,
    "error_page": true,
    "deny": true,
    "allow": true,
    "return": true,
}

if !skipKeys[k] {
    return k + fmt.Sprintf("=%s", val), nil
}
```

Comment (wangzhuzhen) [reply internal/tools/orchestrator/hepa/services/api_policy/impl/impl.go:1404]: changed.

**Pull request merged**